sdk/js/src/sentience.js

```diff
@@ -27,6 +27,68 @@ export default class Sentience {
             return false;
         }
     }
+
+    /**
+     * Fetches all previous verified inference calls
+     *
+     * @param {string} apiKey - Galadriel API Key
+     * @param {number} limit - how many items to return
+     * @param {string} cursor - pagination cursor
+     * @param {string} filter - either "mine" or "all" to filter out results
+     * @returns dict
+     */
+    static getHistory(apiKey, limit, cursor, filter) {
```

Comment:
I feel like this function should be async? Or would that not work?
Something like
```
const response = await fetch(...)
if (!response.ok) {...}
return await response.json()
```